src/styles/partials/components/_step-list.scss

```diff
@@ -37,7 +38,7 @@
       font-size: $h3-font-size;
       font-family: $default-heading-font;
       position: absolute;
-      top: -5px;
+      top: -1px;
       bottom: 0;
```

Comment:
I changed this from -5 to -1px because the icons did not line up after the top one. The 2 and 3 were slightly higher than the text which looked odd. @sdurphy @martypowell

Reply:
i'd say maybe this is -2?

Reply:
done